inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortSourceServiceImpl.java

```diff
@@ -285,8 +290,12 @@ private void parseAll() {
     }
 
     private Map<String, CacheZone> parseCacheZones(
+            String clusterName,
             List<SortSourceStreamSinkInfo> sinkList) {
 
+        Preconditions.checkNotNull(sortClusters.get(clusterName));
```

Comment:
Suggest use the `Preconditions` in manager-common module.

Reply:
Fixed, thx